app/uk/gov/hmrc/excisemovementcontrolsystemapi/controllers/actions/ValidateErnInMessageActionImpl.scala

```diff
@@ -29,19 +29,19 @@ import uk.gov.hmrc.excisemovementcontrolsystemapi.utils.EmcsUtils
 import javax.inject.Inject
 import scala.concurrent.{ExecutionContext, Future}
 
-class ValidateErnsActionImpl @Inject()(messageService: MessageService)(
+class ValidateErnInMessageActionImpl @Inject()(messageService: MessageService)(
```

Comment:
can we call this file with the trait name? We should hide the implementation and only refer ti the trait, a less the impl was in a different file. I know there other file named this way, but we can do these other ones another time are are not part of this PR.